modules/lib/lib-node/src/test/resources/test/FindNodesByQueryHandlerTest.js

```diff
@@ -55,6 +55,21 @@ var expectedEmptyJson = {
     'hits': []
 };
 
+var expectedSortJson = {
+    "total": 12902,
+    "count": 2,
+    "hits": [
+        {
+            "id": "b186d24f-ac38-42ca-a6db-1c1bda6c6c26",
+            "score": 1.2300000190734863
+        },
+        {
+            "id": "350ba4a6-589c-498b-8af0-f183850e1120",
+            "score": 1.7000000476837158
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=8807034)
```suggestion
            'score': 1.7000000476837158
```